docs/reference/cat/snapshots.asciidoc

```diff
@@ -1,5 +1,5 @@
 [[cat-snapshots]]
-== cat snapshots
+== Cat single repository snapshots
```

Comment:
I wouldn't create a separate heading for single repo vs multi repo, but instead revise the existing docs

Reply:
Not exactly, re-writing the existing docs, I've made it like this https://github.com/elastic/elasticsearch/pull/42090/commits/65825086ca8366f26f8cbe64da371d2273ec3694